docs/user/integration/dynatrace/README.md

```diff
@@ -259,3 +262,33 @@ For easier access from the Kyma dashboard, add links to new navigation under **D
     ```
 
 2. If your Secret has a different name or namespace, then download the file first and adjust the namespace and name accordingly in the 'dataSources' section of the file.
+
+## Use Dynatrace Dashboards
+
+You can view logs, traces, and metrics in Dynatrace dashboards:
```

Comment:
```suggestion
You can view metrics in Dynatrace dashboards:
```
Let's not irritate people so that they think this dashboardas here are about logs